api/specs/webserver/components/schemas/group.yaml

```diff
@@ -0,0 +1,155 @@
+GroupAccessRights:
+  description: defines acesss rights for the user
+  type: object
+  properties:
+    read:
+      type: boolean
+    write:
+      type: boolean
+    delete:
+      type: boolean
+  required:
+    - read
+    - write
+    - delete
+  example:
+    # Member
+    - read: true
+      write: false
+      delete: false
+    # Manager
+    - read: true
+      write: true
+      delete: false
+    # Administrator
+    - read: true
+      write: true
+      delete: true
+
+UsersGroup:
+  type: object
+  properties:
+    gid:
+      description: the group ID
+      type: string
```

Comment:
Based on the examples, shouldn't we put ``integer`` instead of ``string``?

Reply:
hmm maybe yes, actually I continued using the same that was done with the user id that is also an integer in the DB and a string in the frontend.
I fear a bit of changing this right now that will trigger a bunch of problems in the testing and whatever else.